src/lib/connection.ts

```diff
@@ -1,14 +1,31 @@
+import { Server as AsyncAPIServer, AsyncAPIDocument } from "@asyncapi/parser"
+
+interface IGleeConnectionConstructor {
+  connection: any,
+  channels: string[],
+  serverName: string,
+  server: AsyncAPIServer,
+  parsedAsyncAPI: AsyncAPIDocument,
+}
+
 class GleeConnection {
+  public rawConnection: any
+  public channels: string[]
+  public serverName: string
+  public AsyncAPIServer: AsyncAPIServer
+  public parsedAsyncAPI: AsyncAPIDocument
```

Comment:
I'll make this stuff private in the next PR. I want to leave it public so it behaves the same way as in JS.

Reply:
https://github.com/asyncapi/glee/pull/213/files#diff-eed0072029be2b474fc4e5968e94834051b17ed462298d5e5e22e9784b51fd45R12-R16